primitives/state-machine/Cargo.toml

```diff
@@ -34,8 +34,10 @@ tracing = { version = "0.1.29", optional = true }
 [dev-dependencies]
 hex-literal = "0.3.4"
 sp-runtime = { version = "4.0.0-dev", path = "../runtime" }
+sp-io = { version = "4.0.0-dev", path = "../io" }
```

Comment:
I missed this last time, but do you think it would be doable to move the tests that uses sp-io to another crate (sp-io depends on sp-state-machine so this will block crate publishing).

Reply:
right, will make the tests work without them.

Reply:
had to move it to sp-runtime to avoid circular deps in the end :\